### PR TITLE
update unmatching model codes

### PR DESCRIPTION
**Pull request by PineApple777**
수정 사항
- in_channel, out_channel이 기존 코드 작성자의 모델에서는 서로 다른 크기인데, 이를 반영함
- SeparableConv2d 부분이 기존 코드 작성자의 모델과 일치하지 않는 문제 수정 (channel, groups etc)
- model.py에서 batchnormalization 부분이 반복해서 쓰이는 문제, 필요한 부분마다 별도로 추가해줘야 한다.
- model.summary() 코드 추가 (전체 모델 확인 용의)

### CODE REVIEW

**Comment by PineApple777** [comment]
위와 같이 기존 Randomly Wired NN 모델과 일치하지 않고, filter라는 값으로 in_channel 과 out_channel을 동일하게 가져가는 문제점이 있었습니다. 이 모델은 in_channel과 out_channel이 동일하지 않습니다. 이부분 확인해보시기 바랍니다.
추가로 batchnormalization 부분은 중복된다고 한번 선언하고 재사용하시면 안됩니다. 이부분도 학습시 적용되는 부분이라, 기존 코드 작성자가 사용한 것 처럼 동일하게 필요한 부분에 작성해주셔야 합니다. 모양도 달라서 재사용 불가능합니다.

현재 제가 테스한 결과 학습 정확도는 약 94% 테스트 정확도는 약 84% 인데, 추가로 최유철 연구원께서 테스트 해보시기 바랍니다. 제가보기에는 CrossEntropy 부분이 특이한 것을 사용한 것 같은데, 코드 원작자의 사용법을 확인해보시고 학습이 올바르게 되는지 검증 부탁드립니다.
